### ISSUE TITLE
Docker for windows 10 home edition

### ISSUE THREAD

**Issue by himarshaj** [cs531-f19/discussions#21]
Hello,

It is mentioned that : Docker Desktop requires Windows 10 Pro or Enterprise version 15063 as a prerequisite. 

So i guess docker is not supporting the windows 10 home edition ?

**Comment by ibnesayeed** [comment]
Yes, Docker's recommended desktop version does not support Windows 10 Home Edition. However, there is a legacy version called [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/) which might work. If I remember correctly, this legacy version installs VirtualBox on Windows and runs a version of Linux inside to enable Docker support.

My personal recommendation would be to not use Windows for development of this course's assignments, but if one insists on using Windows as their primary operating system then at least setup a Linux virtual machine and work inside of it.

**Comment by himarshaj** [comment]
Thank you. My personal computer is Windows 10 Home Edition. I might go with your last option and set up a linux virtual machine to work inside of it when at home.

**Comment by ibnesayeed** [comment]
Also, if you are not afraid of working in the terminal, you can SSH to your office computer and work remotely.
